src/pydicom/cli/main.py

```diff
@@ -39,14 +46,22 @@
     "Examples: "
     "path/to/your_file.dcm, "
     "your_file.dcm::StudyDate, "
+    "your_file.dcm::(0001,0001), "
     "pydicom::rtplan.dcm::BeamSequence[0], "
-    "yourplan.dcm::BeamSequence[0].BeamNumber"
+    "yourplan.dcm::BeamSequence[0].BeamNumber, "
+    "pydicom::rtplan.dcm::(300A,00B0)[0].(300A,00B6)"
 )
 
 
 def eval_element(ds: Dataset, element: str) -> Any:
+    if element[0] != ".":
+        element = "." + element
+
+    # replace all ".(gggg,eeee) hex tags with eval'uable expression"
```

Comment:
Nitpicking comment again - the apostroph should probably be after the tag expression

Reply:
Yep - makes sense